client/components/NonIdealState/NonIdealState.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
```

Comment:
Though I understand why Lighthouse complains about this, it ultimately makes me sort of sad because the `NonIdealState` h4's are useful landmarks in practice for navigating PubPub with a screenreader.

Reply:
Yeah, I noticed that too while using Voice Over. Unfortunately, Lighthouse complains about it and it appears as though an increasing number of folks are using that as their test suite and require 100s. I'll make the case that this is not actually an improvement and see if we can revert later, but I'm not hopeful.